datasync/chaindatafetcher/config.go

```diff
@@ -0,0 +1,27 @@
+package chaindatafetcher
+
+//go:generate gencodec -type ChainDataFetcherConfig -formats toml -out gen_config.go
+type ChainDataFetcherConfig struct {
+	EnabledChainDataFetcher bool
+	NoDefaultStart          bool
+	NumHandlers             int
+	JobChannelSize          int
+	BlockChannelSize        int
+
+	DBHost     string
+	DBPort     string
+	DBName     string
+	DBUser     string
+	DBPassword string
+}
+
+var DefaultChainDataFetcherConfig = &ChainDataFetcherConfig{
+	EnabledChainDataFetcher: false,
+	NoDefaultStart:          false,
+	NumHandlers:             10,
+	JobChannelSize:          50,
+
+	BlockChannelSize: 500,
+
+	DBPort: "3306",
+}
```

Comment:
How about using these values for initial flag values in `cmd/utils/flags.go`?

Reply:
Applied. 5cd2c1301d87de8d6fe5a90a7d959cb13915867a